### ISSUE TITLE
Invisible image being read

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#5838]
a11y severity: minor

SurveyGizmo injects “sg-header-image” which is read as “image” to screenreaders without context when no image is actually shown. This should be removed ideally (maybe an option in SurveyGizmo?) or given CSS of `display: none`.

┆Issue is synchronized with this [Jira Bug](https://jira.mozilla.com/browse/FXA-2222)
┆Issue Number: FXA-2222
